AnnoyancesFilter/sections/annoyances.txt

```diff
@@ -1890,6 +1890,7 @@ gigadownloads.co,techsini.com##.onp-sl-overlap-box
 warezcrack.net#$#.onp-sl-content > [data-lock-id^="onpLock"] { display: block!important; }
 djcracx.com#$#.onp-locker-call { display: block !important; }
 !
+xsplit.com##.smartbanner
```

Comment:
I'm not sure if this element should be hidden, but if it should be hidden then it requires additional rule to fix `.nav` element.
<details><summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/29142494/60982168-e90d4300-a337-11e9-975c-00a8905fea10.png)

</details>

Reply:
I guess it should be blocked as popup, and `body` and `.nav` should be moved upper